tcms/testcases/migrations/0006_merge_text_field_into_testcase_model.py

```diff
@@ -37,6 +37,10 @@ def forward_copy_data(apps, schema_editor):
             history.save()
 
 
+def backward_empty_callable(apps, schema_editor):
+    pass
```

Comment:
Note: going backwards this will leave the text fields empty.

Reply:
Which fields exactly? I'm a little confused on what's happening here.